sass/color/_contrast.scss

```diff
@@ -138,13 +140,13 @@ $WCAG-CONTRAST: (
   $color: map-get($contrast, 'color');
 
   @if ($name != $color) and not is-private-token($name) {
-    $name: _a_var-name($name, $color-var-prefix);
+    $name: dashed-ident($name, $color-var-prefix);
     @return var($name, $color);
   }
 
   @return $color;
 }
-@include _a_register-function('var-contrast');
+@include register-internal('var-contrast');
```

Comment:
just out of curiosity, where is this defined?

Reply:
This was probably a global search-and-replace that isn't quite accurate. But again, it's in the color module which I'm working on in a separate branch - so not worried about it here…